.agent/skills/takt-analyze/SKILL.md

```
@@ -0,0 +1,162 @@
+**手順:**
+1. カスタムファセットの内容をビルトインファセットと比較
+2. 類似度が高い場合はビルトインへの置き換えを提案
+3. ビルトインのbare name参照とセクションマップ参照の混在を検出
+
+## ワークフロー
+
+### Step 1: 対象の特定
+
+分析対象のピースYAMLを特定する。
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)